src/github.com/getlantern/lantern-ui/app/js/controllers.js

```diff
@@ -69,15 +69,25 @@ app.controller('RootCtrl', ['$rootScope', '$scope', '$compile', '$window', '$htt
       localStorageService.set($rootScope.lanternHideMobileAdVar, true);
     };
 
+    $rootScope.mobileAppLink = function() {
+      return "https://bit.ly/lanternapk";
```

Comment:
Controller is not the best place to put constants, but `js/constants.js` is nearly obsolete. I just don't want to touch that file